src/main/java/core/basesyntax/util/FileReaderImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.util;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FileReaderImpl implements FileReader {
+
+    @Override
+    public List<String> read(String filePath) {
+        try {
+            return Files.lines(Path.of(filePath))
+                    .collect(Collectors.toList());
+        } catch (IOException e) {
+            throw new RuntimeException("Cannot read file by path " + filePath);
```

Comment:
The exception message in the catch block should include more details about the exception to aid in debugging. Consider logging the exception or providing more context in the error message.